n6001/hardware/ofs_n6001/build/afu_ip.qsf

```diff
@@ -3,11 +3,9 @@
 
 set_global_assignment -name SOURCE_TCL_SCRIPT_FILE bsp_design_files.tcl
 
-#increase placement and routing effort (at the expense of extra time)

```

Comment:
Remove obsolete settings; add the MPF/VTP stuff that used to be inserted during run.sh but might as well be done here.